src/main/java/org/pesmypetcare/webservice/dao/medalmanager/UserMedalDaoImpl.java

```diff
@@ -70,6 +80,23 @@ public Object getField(String owner, String name, String field) throws DatabaseA
         return dbDoc.getDocumentField(medalPath, field);
     }
 
+    /**
+     * Create all user medals when user is created.
+     * @param username Username of the user.
+     * @throws DatabaseAccessException When the retrieval is interrupted or the execution fails
+     * @throws DocumentException When the document does not exist
+     */
+    public void createAllUserMedals (String username) throws DatabaseAccessException, DocumentException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['(' is preceded with whitespace.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5630992)